.travis.yml

```diff
@@ -50,18 +53,7 @@ git:
   submodules: false
 addons:
   apt:
-    sources:
-    - ubuntu-toolchain-r-test
-    - sourceline: ppa:libreoffice/libreoffice-4-2
-    - llvm-toolchain-trusty-6.0
-    packages:
-    - bsdtar
-    - doxygen
-    - doxygen-doc
-    - doxygen-latex
-    - doxygen-gui
-    - graphviz
-    - xmlstarlet
```

Comment:
Wait a second: `xmlstarlet` is needed during release to generate automatically the XML choco package updates. I am not sure that it will be present by default in the distro. Can you please add it back?